stacker/util.py

```diff
@@ -607,10 +637,9 @@ def extension():
         return '.zip'
 
 
-class SourceProcessor(object):
-    """Makes remote python package sources available in current environment."""
-
-    ISO8601_FORMAT = '%Y%m%dT%H%M%SZ'
```

Comment:
Looks like an error in rebasing here -- this class variable shouldn't go away.

Reply:
update to fix this... a little scary that no tests caught this though.  Makes me think this could use some more tests w/ stubber.  I'll take a poke at that in a bit.